src/openms_gui/source/VISUAL/DIALOGS/ToolsDialog.cpp

```diff
@@ -66,12 +68,15 @@ namespace OpenMS
           String ini_file,
           String default_dir,
           LayerDataBase::DataType layer_type,
-          String layer_name
-    ) :
-    QDialog(parent),
-    ini_file_(ini_file),
-    default_dir_(default_dir),
-    params_(params)
+          const String& layer_name,
+          TVToolDiscovery* tool_scanner) :
+          QDialog(parent),
+          ini_file_(std::move(ini_file)),
+          default_dir_(std::move(default_dir)),
```

Comment:
I'd caution against this `move`, because accidental usage of `ini_file` will cause trouble.